### ISSUE TITLE
Git clone failed for Inference Engine: errror or ?

### ISSUE THREAD

**Issue by kubilaykilinc** [MoonRide303/Fooocus-MRE#142]
Hi,

When i start FooocurMRE, i am getting this message while loading
"Git clone failed for Inference Engine: failed to lock file 'E:/Fooocus-MRE/repositories/ComfyUI-from-StabilityAI-Official/.git/refs/remotes/origin/master.lock' for writing:"

Is this problem or usual process?

Thanks

**Comment by MoonRide303** [comment]
@kubilaykilinc No, it shouldn't happen. It seems like another application is trying to do something with ComfyUI repository. Verify if you don't have any additional application doing something with that folder (or maybe launched Fooocus twice). You can also try removing repositories folder and unzipping it again from release file.
